main.go

```diff
@@ -423,6 +445,16 @@ func run(c *cli.Context) error {
 		return fmt.Errorf("Error: %s\n", err)
 	}
 
+	// Apply Kubernetes secrets manifests
+
+	argsSecret := applyArgs(c.Bool("dry-run"), manifestsSecret)
```

Comment:
There is no check in case the Secret manifest is not included (it is an optional field/file).

Reply:
Check added